src/cms/constants/administrative_division.py

```diff
@@ -1,3 +1,35 @@
+"""
+This module contains constants which represent all possible administrative divisions of a region in Germany:
+
+* ``CITY``: City
+
+* ``DISTRICT``: District
+
+* ``RURAL_DISTRICT``: Rural district
+
+* ``REGION``: Region
+
+* ``CITY_AND_DISTRICT``: City and district
+
+* ``URBAN_DISTRICT``: Urban district
+
+* ``GOVERNMENTAL_DISTRICT``: Governmental district
+
+* ``CITY_STATE``: City state
+
+* ``AREA_STATE``: Area state
+
+* ``FREE_STATE``: Free state
+
+* ``FEDERAL_STATE``: Federal state
+
+* ``MUNICIPALITY``: Municipality
+
+* ``COLLECTIVE_MUNICIPALITY``: Collective municipality
+
+* ``INITIAL_RECEPTION_CENTER``: Initial reception center
```

Comment:
Wow, these are really informative descriptions :stuck_out_tongue_closed_eyes:

Reply:
Do you think the equivalent terms for Germany would help here?